package.json

```diff
@@ -55,9 +55,12 @@
   "files": [
     "build/playcanvas.js",
     "build/playcanvas.min.js",
-    "build/playcanvas.mjs",
     "build/playcanvas.dbg.js",
     "build/playcanvas.prf.js",
+    "build/playcanvas.mjs",
+    "build/playcanvas.min.mjs",
```

Comment:
I would argue we shouldn't ship this. Devs will use the module in their own build process which will do it's own bundling/minification.

Reply:
Yeah I agree it might have limited usefulness, but is that reason enough to remove it from the package? For example couldn't users in future load the mjs from the browser directly, in which case the min version might be useful?

Reply:
They could load the module build directly from a script tag but I don't see that as being a common use case. I guess there's no major harm to including it. But I might suggest removing it in future. 😄